.ci/setup-laravel-dev-fixture.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env sh
+
+set -e
+
+cd features/fixtures
+
+rm -rf laravel-latest
+
+# Ignore dev dependencies because we don't need them to run the Maze Runner tests
+# and they will only introduce more failure points
+composer create-project laravel/laravel laravel-latest --no-dev
+
+cd laravel-latest
+
+composer require 'laravel/framework:dev-master as 7' --update-with-dependencies --no-update
```

Comment:
dev-master is a bad idea, especially if that branch gets renamed to 8.x in the future, or something.

Reply:
Just ask composer for `8.*@dev`.

Reply:
Is that actually the branching strategy used by Laravel? I don't want to test v8 here, I want to test whatever is the most cutting edge

Reply:
It's impossible to predict the future, but I'd say its safer to write 8.*, and when development on 9.* starts, we can add that too. I think that's preferable to actually testing the latest version. Why? Because the moment when master starts becoming 9.x dev, we have no test coverage for 8.x, and this is exactly when people still start upgrading to 8.x and exactly when there might be last minute breaking changes made to 8.x, so certainly better to be testing 8.x on 8.0.0 release day and not testing 9.x for a few weeks, than to miss out on 8.x breaking at exactly the time when we needed to know.

Reply:
I would tend to agree with Graham here, we should be conscious of when releases are being made.  It'll not only be practical but also act as a mental check when we decide we support a new version.

Reply:
> the moment when master starts becoming 9.x dev, we have no test coverage for 8.x, and this is exactly when people still start upgrading to 8.x and exactly when there might be last minute breaking changes made to 8.x, so certainly better to be testing 8.x on 8.0.0 release day and not testing 9.x for a few weeks, than to miss out on 8.x breaking at exactly the time when we needed to know

I don't really agree; we don't support the next major until it's also passing on CI, so there's never a time when someone using Bugsnag can upgrade before we have test coverage for it and if the tests break then we need to fix them before we can support it anyway. If we're not happy with our test coverage to be confident then that's a separate issue (and I manually test new releases for this reason)

Bear in mind this doesn't run on every commit — it will run only on a cron and merges to master/next — so if it starts failing it doesn't block merging branches. The point isn't to test if every change is compatible, it's to give an early warning that we might have to make changes to support the upcoming version

> I would tend to agree with Graham here, we should be conscious of when releases are being made. It'll not only be practical but also act as a mental check when we decide we support a new version.

We already have to be conscious of new releases because we have to explicitly support them in our `composer.json`, so they get added to CI then. Making this point at a branch other than master just means another place to update at the same time

I think we should be adding maze runner fixtures for every major Laravel version that we support (or at least the versions that still get updates), but that's a separate task to this piece of work

Reply:
I've also just pushed 4571121d55f73d7bea33e98247f25a33cb066ce9 so these tests _only_ run via cron — i.e. they will never block merging branches or releasing